src/pages/Items.tsx

```diff
@@ -6,26 +6,29 @@ import ItemList from '../components/ItemList';
 import Button from '../components/Button';
 import Dropdown from '../components/DropdownSort';
 import Pagenation from '../components/Pagenation';
+import type { Product } from '../api/apiTypes';
 import { getProducts } from "../api/api";
 import icSearch from '../assets/ic_search.svg';
 import './Items.css';
 
-const PAGESIZE = {
+const PAGESIZE: { [size: string]: number; } = {
```

Comment:
💬 여담
이런 경우 Record 유틸리티 타입을 이용해 더 간단히 작성하실 수 있습니다.

```suggestion
const PAGESIZE: Record<string, number>= {
```